lib/fakeroot.py

```diff
@@ -224,22 +245,46 @@
      "cmds": ["apt", "apt-get", "dpkg"],
      "each": ["fakeroot"] },
 
-   # Fedora notes:
-   #
-   # 1. The minimum supported version was chosen somewhat arbitrarily based on the
-   #    release versions available for testing (what was on Docker Hub).
-   #
-   # 2. The fakeroot package is in the base repository set so enabling EPEL is
-   #    not required.
-
-   "fedora":
-   { "name": "Fedora 24+,",
-     "match":  ("/etc/fedora-release", r"release (?!1?[0-9] |2[0-3] )"),
+   "suse":
+   { "name": "(Open)SUSE 42.2+",  # no fakeroot before this
+     # I don't know if there are OpenSUSE derivatives
+     "match": ("/etc/os-release", r"ID_LIKE=.*suse"),
      "init": [ ("command -v fakeroot > /dev/null",
-                "dnf install -y fakeroot") ],
-     "cmds": ["dnf", "rpm", "yum"],
+                # fakeroot seems to have a missing dependency, otherwise
+                # failing with missing getopt in the fakeroot script.
+                "zypper refresh; zypper install -y fakeroot /usr/bin/getopt") ],
+     "cmds": ["zypper", "rpm"],
      "each": ["fakeroot"] },
 
+   # no worky; see #1295
+   # [It seems to me (Dave Love) that it works sufficiently well, just
```

Comment:
```suggestion
   # It seems to me (Dave Love) that it works sufficiently well, just
```
Typo?

Reply:
There's a closing bracket too so I think this was intended.